fluentd/configs.d/user/fluent.conf

```diff
@@ -17,6 +17,7 @@
 <label @INGRESS>
 ## filters
   @include configs.d/openshift/filter-pre-*.conf
+  @include configs.d/openshift/filter-force-utf8.conf
```

Comment:
This fluent.conf isn't used.  The source is in openshift-ansible: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging_fluentd/templates/fluent.conf.j2

However, you could make this work by creating this filter as a new file e.g. `configs.d/openshift/filter-pre-force-utf8.conf` which will be picked up by the wildcard match `@include configs.d/openshift/filter-pre-*.conf`

Reply:
Done.